.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: ruby
+sudo: false
+addons:
+  apt:
+    packages:
+    - libgmp-dev
+    - libmpfr-dev
+    - libmpc-dev
+    - binutils-dev
+rvm:
+ - 1.9.3
+ - 2.0
+ - 2.1
+
+install:
+  - export RUBY_SOURCE_DIR=`pwd`
+  - export TEST_CPP="no"
+
+  - git clone https://github.com/symengine/symengine symengine-cpp
+  - cd symengine-cpp
+  - git checkout 8537c67507fae9b1d8d575d322c6ade4f1a59297
```

Comment:
Right. Eventually we'll use a release tag, but right now we are just trying to set it up.